backend/src/main/java/com/redhat/cloud/notifications/auth/ConsoleIdentityProvider.java

```diff
@@ -134,16 +142,25 @@ public Uni<SecurityIdentity> authenticate(ConsoleAuthenticationRequest rhAuthReq
                     .addRole(adminRole)
                     .build());
         }
+
         // Retrieve the identity header from the authentication request
-        return Uni.createFrom().item(() -> (String) rhAuthReq.getAttribute(X_RH_IDENTITY_HEADER))
-                .onItem().transformToUni(xRhIdHeader -> buildQuarkusUni(xRhIdHeader));
+        return Uni
+            .createFrom()
+            .item(
+                () -> (String) rhAuthReq.getAttribute(X_RH_IDENTITY_HEADER)
+            )
+            .onItem()
+            .transformToUni(this::buildQuarkusUni);
     }
 
     @CacheResult(cacheName = "rbac-cache")
     protected Uni<QuarkusSecurityIdentity> buildQuarkusUni(String xRhIdHeader) {
         // Start building a QuarkusSecurityIdentity
-        return Uni.createFrom().item(QuarkusSecurityIdentity.builder())
-            .onItem().transformToUni(builder -> {
+        return Uni
+            .createFrom()
+            .item(QuarkusSecurityIdentity.builder())
+            .onItem()
+            .transformToUni(builder -> {
```

Comment:
I haven't changed anything here. I just expanded the code to improve its readability.